src/wp-admin/includes/dashboard.php

```diff
@@ -5,7 +5,6 @@
  * @package ClassicPress
  * @subpackage Administration
  */
-
```

Comment:
I'm not sure about these deleted empty lines. Seems like unnecessary changes.

Reply:
Fixed